spyder/plugins/switcher/widgets/switcher.py

```diff
@@ -93,15 +93,51 @@ class Switcher(QDialog):
       SwitcherItem:      [title description    <shortcut> section]
     """
 
-    # Dismissed switcher
+    # --- Signals
     sig_rejected = Signal()
-    # Search/Filter text changes
+    """
+    This signal is emitted when the plugin is dismissed.
+    """
+
     sig_text_changed = Signal(TEXT_TYPES[-1])
```

Comment:
```suggestion
    sig_text_changed = Signal(str)
```

And please don't forget to remove the `TEXT_TYPES` import above.